src/Amadeus/Client/RequestCreator/Converter/Service/StandaloneCatalogueConv.php

```diff
@@ -0,0 +1,45 @@
+<?php
+/**
+ * amadeus-ws-client
+ *
+ * Copyright 2015 Amadeus Benelux NV
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * @package Amadeus
+ * @license https://opensource.org/licenses/Apache-2.0 Apache 2.0
+ */
+
+namespace Amadeus\Client\RequestCreator\Converter\Service;
+use Amadeus\Client\RequestCreator\Converter\BaseConverter;
+use Amadeus\Client\RequestOptions\ServiceStandaloneCatalogueOptions;
+use Amadeus\Client\Struct;
+
+/**
+ * Service_StandaloneCatalogue Request converter
+ *
+ * @package Amadeus\Client\RequestCreator\Converter\Service
+ * @author Arvind Pandey <arvindpandey87@gmail.com>
+ */
+class StandaloneCatalogueConv extends BaseConverter
+{
+    /**
+     * @param ServiceIntegratedCatalogueOptions $requestOptions
+     * @param int|string $version
+     * @return Struct\Service\StandaloneCatalogue
+     */
+    public function convert($requestOptions, $version)
+    {
+        return new Struct\Service\StandaloneCatalogue($requestOptions);
+    }
+}
```

Comment:
According to the PSR-2 standard, there has to be an empty line following the closing bracket at the end of the file.